models/users.js

```diff
@@ -622,6 +622,25 @@ const removeGitHubToken = async (users) => {
     throw err;
   }
 };
+
+const getUsersByRole = async (role) => {
+  try {
+    const usersRef = await userModel.where(`roles.${role}`, "==", true).get();
```

Comment:
What do we get here is role is invalid? Error or empty array?

Reply:
throw an error which is handled with 5xx response.

Reply:
in the test you can see that i have entered a random role and i check for the error to occur.